### PR TITLE
docs: updating commit lint

### PR DESCRIPTION
**Pull request by GeronimoJohn**
# Description

Linting is currently failing on main

Failing PR - #707 (270cf47)

Earliest good commit (used as reference) - #709 (5364963)

- Link to Basecamp Todo

# How should this PR be QA Tested?

- [ ] Linting should pass

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged into main

### CODE REVIEW

**Review by mikeallisonJS — DISMISSED** [review]

**Review by tataihono — APPROVED** [review]

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
       - name: Prettier
         run: npx prettier --check .
       - name: lint commit messages
-        run: npx commitlint --from aaad3f9 # earliest good commit to main
+        run: npx commitlint --from b147048 # earliest good commit to main
```
Comment (csiyang) [inline .github/workflows/main.yml:42]: I tihnk this needs to be done on another PR

**Review by csiyang — DISMISSED** [review]

**Review by storyworks — APPROVED** [review]
